### DISCUSSION TITLE
Showing chapters

### DISCUSSION THREAD

**Discussion by Bakayalo** [smplayer-dev/smplayer#237] [Ideas]
Right now, the chapter marks only show on the floating progress bar when I seek, and I can't click on it. It'd be nice to see chapter marks in the regular clickable progress bar, and chapter names like a tooltip upon mouseover.

Using on Win10 with mpv default.

**Comment by ThomasEricB** [comment]
This needs to be implemented sooner rather than later

**Reply by hurricane51** [reply]
"floating progress bar when I seek". Is there a way to skip to subsequent chapter marks, or just to view them?

**Comment by dark-penguin** [comment]
I'd also really like to be able to configure OSD to show chapter names, and some way to quickly go into "Browse - Chapters" menu. Basically, there is no easy way to see what's the name of the chapter I'm currently watching, and what other chapters are there.

**Comment by apoketo** [comment]
Can there also be something in the chapter_menu dropdown to indicate which chapter you're currently in.

**Comment by KaKi87** [comment]
@smplayer-dev Any news on this please ? :pray:
